configuration/configuration.go

```diff
@@ -57,6 +57,9 @@ type Configuration struct {
 			// contain the private portion for the file specified in
 			// Certificate.
 			Key string `yaml:"key,omitempty"`
+
+			// Specifies the CA cert for client authentication
```

Comment:
This line should point out that these files can contain multiple CA certs, since `AppendCertsFromPEM` parses all certs in the file.